lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -2268,4 +2298,112 @@ Ufe::Path MayaUsdProxyShapeBase::ufePath() const
 
 std::atomic<int> MayaUsdProxyShapeBase::in_compute { 0 };
 
+PcpVariantFallbackMap MayaUsdProxyShapeBase::updateVariantFallbacks(
```

Comment:
Without specifically weighing in about the specific goals of the PR in general, I would like to have these functions to convert to/from text and set the data on a stage into a file in lib/mayaUsd/utils, for example a duo of variantFallbacks.h/variantFallback.cpp. This way they're both reusable and directly testable.

Reply:
Thanks, I'll update it.

Reply:
Hi @pierrebai-adsk ,  I have updated the code based on your feedback.